src/mzn_bench/analysis/report_status.py

```diff
@@ -13,25 +14,14 @@ def status_from_str(s: str) -> Status:
 
 
 def report_status(
-    per_model: bool, per_problem: bool, statistics: Path, avg: str, tablefmt: str
+    keys: Iterable[str], statistics: Path, avg: str, tablefmt: str
```

Comment:
I would lean to a `Enum` type argument, but I suppose with this package `str` is probably fine